pipelines/coref/project.yml

```diff
@@ -0,0 +1,127 @@
+title: "spaCy Coref Model"
+description: >
+  This project trains a coreference model for spaCy using OntoNotes.
+
+vars:
+  # XXX Change to your actual GPU ID
+  gpu_id: 0
+  # XXX fill with your local path to OntoNotes
+  ontonotes: /home/polm/data/ontonotes
+  max_epochs: 5
+
+directories: ["assets", "corpus", "scripts", "configs", "training"]
+
+assets:
+  - dest: "assets/"
+    git:
+      repo: "https://github.com/explosion/conll-2012"
+      branch: "main"
+      path: ""
+    description: "CoNLL-2012 scripts and dehydrated data."
+  - dest: "assets/conll-eval"
+    git:
+      repo: "https://github.com/conll/reference-coreference-scorers"
+      branch: "master"
+      path: ""
+    description: "CoNLL-2012 reference scorer."
+
+workflows:
+  prep:
+    - prep-data
+    - preprocess
+  train:
+    - train-cluster
+    - prep-span-data
+    - train-span-predictor
+    - assemble
+
+  all:
+    # prep
+    - prep-data
+    - preprocess
+    # train
+    - train-cluster
+    - prep-span-data
+    - train-span-predictor
+    - assemble
+
+commands:
+  - name: "prep-data"
+    help: "Rehydrate the data using OntoNotes"
+    script: 
+      # XXX python2 is required for the old scripts
+      - python2 -c "print('Checking for Python 2...')"
+      - tar xzf assets/conll-2012-development.v4.tar.gz -C assets/
+      - tar xzf assets/conll-2012-test-key.tar.gz -C assets/
+      - tar xzf assets/conll-2012-test-official.v9.tar.gz -C assets/
+      - tar xzf assets/conll-2012-train.v4.tar.gz -C assets/
+      #- tar xvzf assets/conll2012-coref/reference-coreference-scorers.v8.01.tar.gz -C assets/
+
+      - echo "Rehydrating data... This will take roughly 20 minutes"
+      - "bash assets/conll-2012/v3/scripts/skeleton2conll.sh -D ${vars.ontonotes} assets/conll-2012"
+
+      # Concatenate the data into single files to make it easier to work with
+      - sh -c "rm -f assets/*._gold_conll"
+      - sh -c "cat assets/conll-2012/v4/data/development/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/dev.gold.conll"
+      - sh -c "cat assets/conll-2012/v4/data/train/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/train.gold.conll"
+      - sh -c "cat assets/conll-2012/v4/data/test/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/test.gold.conll"
```

Comment:
These commands don't seem to work on Windows, because of an issue we have with quotes. I think this might also be the case on Mac, we have to check. The commands themselves are fine though.

Running through `spacy projects`: 

```
Running command: sh -c '"cat assets/conll-2012/v4/data/development/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/dev.gold.conll"'
sh: cat assets/conll-2012/v4/data/development/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/dev.gold.conll: No such file or directory
```

Running directly on console, no issue, and the 3 files are properly created:
```
$ sh -c "cat assets/conll-2012/v4/data/development/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/dev.gold.c
$ sh -c "cat assets/conll-2012/v4/data/train/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/train.gold.conll
$ sh -c "cat assets/conll-2012/v4/data/test/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/test.gold.conll"
```

Reply:
This should be addressed [here](https://github.com/explosion/projects/pull/101/commits/a3814210283d363c80a45a9cd77e199b7dcf6ade).